src/commands/read.ts

```diff
@@ -95,11 +96,12 @@ export const handler = async (argv: Arguments<Options>): Promise<void> => {
           format: "objects",
           unwrap: argv.unwrap,
         });
-      } catch (e: any) {
-        if (e.message.includes("in JSON at position")) {
-          console.log("Can't unwrap multiple rows. Use --unwrap=false");
+      } catch (err: any) {
+        if (err.message.includes("in JSON at position")) {
```

Comment:
We had a mix of caught errors being named `e` and `err`. Changing this to be consistent